site/cypress/support/index.js

```diff
@@ -16,3 +16,7 @@
 import './commands';
 
 require('@cypress/skip-test/support');
+
+afterEach(() => {
+    cy.logout(true);
+});
```

Comment:
Leaving tests in a logged-in state works most of the time but can occasionally be flaky; see https://github.com/Submitty/Submitty/pull/7695#discussion_r818181484 for more details.

I added in a list that lets certain tests be excluded from logging out, though it only excludes based on the full path of the current test title and not the file name itself (Cypress.spec reports differently when running all tests versus running each test individually so I don't think there's a way to check the filename without a hacky workaround, and I don't think there will be a case where there will be two tests with the same title paths).

Reply:
Could we make the logout function have a conditional (activated via parameter) such that it can check if we're logged in first before triggering the action, else do nothing? I think that might be a bit cleaner than using text ignores that could be easy to inadvertently break.